docs/tutorial/part-eight/index.md

```diff
@@ -331,44 +330,45 @@ export const query = graphql`
 `
 ```
 
-The above example is based off the [Gatsby Starter Blog](/starters/gatsbyjs/gatsby-starter-blog/). By passing props to the `<SEO>` component, you can dynamically change the metadata for a post. In this case, the blog post `title` and `excerpt` (if it exists in the blog post markdown file) will be used instead of the default `siteMetadata` properties in your `gatsby-config.js` file.
+उपयुर्क्त उदहारण [Gatsby स्टार्टर ब्लॉग](/starters/gatsbyjs/gatsby-starter-blog/) पर आधारित है। अगर आप `<SEO>` ककौम्पोनॅन्ट मे प्रॉप्स पास करते है, तो आप डाईनामिकली एक पोस्ट की मेटाडाटा बदल सकते हैं। इस मामले में, `gatsby-config.js` फाइल मे डिफ़ॉल्ट `siteMetadata` प्रॉपर्टी के बदले ब्लॉग पोस्ट के `title` और `excerpt` का इस्तेमाल किया जायेगा (अगर वह ब्लॉग पोस्ट के अंदर मौजूद है तोह)।   
 
-Now, if you run the Lighthouse audit again as laid out above, you should get close to--if not a perfect-- 100 score!
+अब, अगर आप उपयुर्क्त तरीके से Lighthouse ऑडिट को फिर से चलाते है, तो आपको १०० स्कोर या १०० स्कोर के बेहद करीब मिलेंगे!
```

Comment:
`१००` => `100`

let's not translate numbers, it might be a bit difficult for some people to understand

Reply:
I can see this `१००` at two places. Can you update this too?